jobs/grafana/templates/create-update-datasources.erb

```diff
@@ -45,23 +26,29 @@ if [[ $exitCode != 0 ]]
 then
   echo -e "\nError: Cannot connect to Grafana after 3 retries."
   exit 1
-fi
+  fi
+
+<%
+  require 'json'
+  defaults = {"access" => "proxy", "basicAuth" => false}
+  datasources = p('grafana.datasources')
+  if !p('grafana.datasource').empty?
```

Comment:
I think this'd be better off as:

```ruby
if_p("grafana.datasource") do |datasource|
  datasources << datasource
end
```

And then removing the `default` from the spec for that property.